### DISCUSSION TITLE
How to hold space and drag the screen even if your mouse is over a node?

### DISCUSSION THREAD

**Discussion by wigen** [retejs/rete#590] [Q&A]
How to hold space and drag the screen even if your mouse is over a node?

**Comment by Ni55aN** [comment]
Let's figure it out: in order to drag the screen, the area (which contains the nodes) receives a `pointerdown` event. Since nodes have their own `pointerdown` and `stopPropagation`, the region doesn't receive this event when the pointer is over the node.

It can be disabled by assigning the `pointer-events: none` property to all nodes within (or even for all children of the area).


```js
document.addEventListener('keydown', (e) => {
    if (e.code === 'Space') {
        editor.view.area.el.classList.add('lock-pointer-events')
    }
})
document.addEventListener('keyup', (e) => {
    if (e.code === 'Space') {
        editor.view.area.el.classList.remove('lock-pointer-events')
    }
})
```

where `lock-pointer-events` is a CSS class:
```css
.lock-pointer-events * {
  pointer-events: none;
}
```
